README.rst

```diff
@@ -17,56 +17,56 @@ __ https://docs.rs/ordermap/
 Background
 ==========
 
-This was inpired by Python 3.6's new dict implementation (which remembers
+This was inspired by Python 3.6's new dict implementation (which remembers
 the insertion order and is fast to iterate, and is compact in memory).
 
 Some of those features were translated to Rust, and some were not. The result
-was ordermap, a hash table that is
+was ordermap, a hash table that has following properties:
 
-- Order is **independent of hash function** and hash values of keys
-- Fast to iterate
-- Indexed in compact space
-- Preserves insertion order **as long** as you don't call ``remove``.
-- Using robin hood hashing just like Rust's libstd HashMap.
+- Order is **independent of hash function** and hash values of keys.
+- Fast to iterate.
+- Indexed in compact space.
+- Preserves insertion order **as long** as you don't call ``.remove()``.
+- Uses robin hood hashing just like Rust's libstd ``HashMap``.
 
   - It's the usual backwards shift deletion, but only on the index vector, so
     it's cheaper because it's moving less memory around.
 
 Does not implement (Yet)
 ------------------------
 
-- ``.reserve`` exists but does not have a full implementation
+- ``.reserve()`` exists but does not have a full implementation
 
 Performance
 -----------
 
-OrderMap derives a couple of performance facts directly from how it is constructed,
+``OrderMap`` derives a couple of performance facts directly from how it is constructed,
 which is roughly:
 
   Two vectors, the first, sparse, with hashes and key-value indices, and the
   second, dense, the key-value pairs.
 
 - Iteration is very fast since it is on the dense key-values.
 - Removal is fast since it moves memory areas only in the first vector,
-  and uses a single swap in the second vector
+  and uses a single swap in the second vector.
 - Lookup is fast-ish because the hashes and indices are densely stored.
   Lookup also is slow-ish since hashes and key-value pairs are stored in
-  separate places. (Visible when cpu caches size is limiting.)
+  separate places. (Visible when cpu caches size is limiting).
```

Comment:
It would be silly of me to disagree but.. shouldn't this not change? The whole sentence is inside the ()'s.

Reply:
I actually wanted something like "Sentence (clarification)." but now I think it's better to leave it as it was before because there are already 2 sentences out of parentheses.

Reply:
Force-pushed the change.